dashboards-observability/public/components/visualizations/charts/shared/helper.ts

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { ConfigTooltip } from '../../../event_analytics/explorer/visualizations/config_panel/config_panes/config_controls';
+
+export const fetchConfigObject = (editor: string, propsOptions: any) => {
```

Comment:
we already have util files as common place. so instead of creating new file, can we move this to existing common file.